modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryTopListenedCategoriesView.kt

```diff
@@ -0,0 +1,156 @@
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP40
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.endofyear.stories.StoryTopListenedCategories
+import au.com.shiftyjelly.pocketcasts.models.db.helper.ListenedCategory
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import au.com.shiftyjelly.pocketcasts.images.R as IR
+
+@Composable
+fun StoryTopListenedCategoriesView(
+    story: StoryTopListenedCategories,
+    modifier: Modifier = Modifier,
+) {
+    val context = LocalContext.current
+    var screenHeight by remember { mutableStateOf(1) }
+    var textFieldHeight by remember { mutableStateOf(1) }
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .onGloballyPositioned {
+                screenHeight = it.size.height
+            },
+        verticalArrangement = Arrangement.Center
+    ) {
+        TextH30(
+            text = "Your Top Categories",
+            textAlign = TextAlign.Center,
+            color = story.tintColor,
+            modifier = modifier
+                .fillMaxWidth()
+                .padding(vertical = 16.dp)
+                .onGloballyPositioned {
+                    textFieldHeight = it.size.height
+                }
+        )
+        story.listenedCategories.subList(0, minOf(story.listenedCategories.count(), 5))
+            .mapIndexed { index, listenedCategory ->
+                CategoryItem(
+                    listenedCategory = listenedCategory,
+                    position = index,
+                    tintColor = story.tintColor,
+                    iconSize = getIconSize(screenHeight, textFieldHeight, context)
+                )
+            }
+    }
+}
+
+@Composable
+fun CategoryItem(
+    listenedCategory: ListenedCategory,
+    position: Int,
+    tintColor: Color,
+    iconSize: Dp,
+    modifier: Modifier = Modifier,
+) {
+    Column {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp)
+        ) {
+            TextP40(
+                text = "${position + 1}",
+                color = tintColor,
+                modifier = modifier.padding(end = 16.dp)
+            )
+            Row(
+                modifier = modifier
+                    .padding(end = 16.dp)
+                    .weight(1f),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Box(modifier = modifier.padding(top = 4.dp, end = 12.dp, bottom = 4.dp)) {
```

Comment:
Just curious, why did you decide to use a `Box` for the padding here instead of applying the padding to the `Image`'s modifier.

Reply:
[Copy-Paste](https://github.com/Automattic/pocket-casts-android/blob/main/modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/PodcastItem.kt#L46-L51) from `PodcastItem`.
Fixed in [4a1c204](https://github.com/Automattic/pocket-casts-android/pull/531/commits/4a1c2042159496829c87f0ffd3d01c858fedbbf1) and [5394e07](https://github.com/Automattic/pocket-casts-android/pull/531/commits/5394e075cefeba8fdc20957147d1708f68f59e41)